Engine/Log/Log.cpp

```diff
@@ -0,0 +1,85 @@
+#include "Main/Globals.h"
+#include "Main/Application.h"
+#include "Filesystem/Path.h"
+#include "Log/EngineLog.h"
+#include <mutex>
+
+void LittleOrionLogEntry(const EngineLog::LogEntryType type, const char file[], int line, const char* format, ...)
```

Comment:
Just asking, what does it means to put ... as parameter?

Reply:
It's varying list of arguments, it means that you can pass any number of parameters after format